src/Infrastructure/VM/src/ESMCI_VM.C

```diff
@@ -11,7 +11,6 @@
 //==============================================================================
 #define ESMC_FILENAME "ESMCI_VM.C"
 //==============================================================================
-#define GARBAGE_COLLECTION_LOG_off
```

Comment:
YES! I'm so happy to see preprocessor macros removed if they are not architecture/compiler/library dependendent.

Reply:
Same here.